FrameworkSystem/Client/ProxyUpload.py

```diff
@@ -26,7 +26,7 @@ class CLIParams(object):
 
   def __str__(self):
     data = []
-    for k in ('proxyLifeTime', 'diracGroup', 'certLoc', 'keyLoc', 'proxyLoc',
+    for k in ('proxyLifeTime', 'certLoc', 'keyLoc', 'proxyLoc',
```

Comment:
I will not repeat it further, but here as well, the `diracGroup` is removed, which is maybe a bit prejudicial to other users ?

Reply:
DIRAC groups will be added when proxy will be requested

Reply:
That seems to be a terrible security flaw ! Up to now, you can retrieve only proxies that were uploaded. Now, you would be able to get proxies from any group (including Admin) a user belongs to

Reply:
If we allow this for users with no certificates, there is no point not to allow this for the users having the certificates. If somebody has a FullDelegation right, then this is the right to get any proxy. Having said that, we can consider also to forbid downloading proxies with certain groups. Or proxies with groups with certain properties. Practically, this mechanism is only needed to download NormalUser and Pilot proxies. We can enforce that.